recipes/libyuv/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "1767":
```

Comment:
how is the version defined? I there is no release for such version we should have to stick to this https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
libyuv version is defined in [README.chromium](https://chromium.googlesource.com/libyuv/libyuv/+/refs/heads/master/README.chromium) and [version.h](https://chromium.googlesource.com/libyuv/libyuv/+/refs/heads/master/include/libyuv/version.h)

Reply:
This version is not increased in every commit I assume. So we take the commits when the version number in those files get increased?

Is that reliable?

Reply:
I agree with Croydon, maybe this not the best approach. We could use the commit and tag the date if that version is not the official used by the upstream and users.